src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/PopulateUpdateSubmission.java

```diff
@@ -151,5 +153,5 @@ private void populateFilingForDate(
                             .ifPresent(updateSubmission::setFilingForDate);
                 });
     }
-
+    Function <Integer, String> toStringDate = value -> String.format("%02d",value);
```

Comment:
Can this be a local variable? And maybe find a better name as it is not a full date what it formats?

Reply:
Oops snap haha